<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>videoForm</class>
 <widget class="QWidget" name="videoForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>371</width>
    <height>825</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <property name="styleSheet">
   <string notr="true">#videoForm {
                     border-image: url(:/res/MIX3-v.png) 15px 67px 14px 19px;
                     border-width: 15px 67px 14px 19px;
                 }</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>15</number>
   </property>
   <property name="topMargin">
    <number>67</number>
   </property>
   <property name="rightMargin">
    <number>14</number>
   </property>
   <property name="bottomMargin">
    <number>19</number>
   </property>
   <item>
    <widget class="QWidget" name="loadingWidget" native="true">
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">#loadingWidget {
	background-color: red; border: 1px solid black; border-radius: 10px;
}</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QQuickWidget" name="quickWidget">
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">#quickWidget{
background-color: red; border: 1px solid black; border-radius: 10px;
}</string>
        </property>
        <property name="resizeMode">
         <enum>QQuickWidget::SizeRootObjectToView</enum>
        </property>
        <property name="source">
         <url>
          <string>qrc:/qml/pinwheel.qml</string>
         </url>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QYUVOpenGLWidget" name="videoWidget" native="true">
     <property name="styleSheet">
      <string notr="true">#videoWidget{
background-color: red; border: 1px solid black; border-radius: 10px;
}</string>
     </property>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>40</y>
        <width>59</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QQuickWidget</class>
   <extends>QWidget</extends>
   <header>QtQuickWidgets/QQuickWidget</header>
  </customwidget>
  <customwidget>
   <class>QYUVOpenGLWidget</class>
   <extends>QWidget</extends>
   <header>qyuvopenglwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
